src/registry/ecr.go

```diff
@@ -167,82 +225,23 @@ func (r *RegistryScan) GetScanFindings(ctx context.Context, digestInfo RegistryI
 			break
 		}
 
-		if !pg.HasMorePages() {
-			imageScanFindings = *page.ImageScanFindings
-		}
-
-		findings = append(findings, page.ImageScanFindings.Findings...)
-		enhancedFindings = append(enhancedFindings, page.ImageScanFindings.EnhancedFindings...)
-	}
-
-	imageScanFindings.Findings = findings
-	imageScanFindings.EnhancedFindings = enhancedFindings
-
-	return &ScanFindings{
-		ImageScanFindings: imageScanFindings,
-	}, nil
-}
-
-// Filters
```

Comment:
Moved to `finding` package to avoid circular dependency when `Summary` struct was introduced